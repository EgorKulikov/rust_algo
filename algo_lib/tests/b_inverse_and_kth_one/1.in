5 7
1 1 3
2 1
1 0 2
2 0
2 1
1 0 5
2 2
