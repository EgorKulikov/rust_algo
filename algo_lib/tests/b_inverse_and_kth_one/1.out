2
0
2
4
